android/app/src/fdroid/play/listings/fr-FR/release-notes.txt

```diff
@@ -1,12 +1,12 @@
-• Données OpenStreetMap datant du 14 août
-• Nouveaux articles Wikipedia
-• Correction de recherche d'adresses : République tchèque, Slovaquie
-• Ajout d'isolignes pour des régions : d'Argentine, Chili, Kazakhstan, Tibet, USA, Russie
-• Isolignes plus denses dans certaines régions
-• Correctiond de bug sur Nexus 5X
-• Ajout d'itinéraires de ferry
-• Icônes de parking redessinées
-• Priorités remanié pour les POI
-• Beaucoup d'autres améliorations et corrections de bugs
+• New OpenStreetMap maps data as of September 20
+• New Ruler (aka Helicopter) routing for straight distance measurement
+• Background voice and directions
+• Fixed "Wait…" dialog on Android 13 with SD card
+• Do not pause music when the screen is rotated
+• Added and improved many icons and styles
+• Improved search results relevance
+• Parking entrances are searchable
+• Test TTS voice in the settings
+• Fixed arrow position button on Android 5&6
```

Comment:
```suggestion
• Nouvelles données OpenStreetMap du 20 Septembre
• Ajout du mode règle/Hélicoptère pour mesurer la distance entre 2 points
• Support du guidage gps et vocal en arrière plan
• Correction d'un bug sur les téléphones avec carte SD sous Android 13
• La musique n'est plus mis en pause, quand on tourne le téléphone
• Ajout et améliorations de la carte et de nombreuses icônes
• Amélioration de la pertinence des résultats de recherche
• Les entrées de parking peuvent être recherchées
• Ajout d'un bouton dans les paramètres pour tester la synthèse vocale
• Correction de l'affichage de la flèche de position sur Android 5&6
• Améliorations des traductions Arabe, Polonaise et Espagnole
```

Reply:
I can manually update release notes in Google Play for Android, but it's too late for FDroid and App Store :(